### PR TITLE
requestedCapabilities is optional (update connect account)

### PR DESCRIPTION
**Pull request by Maxim-Inv**
https://stripe.com/docs/api/accounts/update?lang=ruby#update_account-requested_capabilities

### CODE REVIEW

File: Sources/StripeKit/Connect/Accounts/AccountRoutes.swift [hunk 1]
```diff
@@ -294,10 +294,14 @@ public struct StripeConnectAccountRoutes: AccountRoutes {
                        externalAccount: Any?,
                        individual: [String: Any]?,
                        metadata: [String: String]?,
-                       requestedCapabilities: [String],
+                       requestedCapabilities: [String]?,
                        settings: [String: Any]?,
                        tosAcceptance: [String: Any]?) -> EventLoopFuture<StripeConnectAccount> {
-        var body: [String: Any] = ["requested_capabilities": requestedCapabilities]
+        var body = [String: Any]()
```
Comment (Andrewangeta) [inline Sources/StripeKit/Connect/Accounts/AccountRoutes.swift]: Can we use the explicit type when declaring this variable to follow the style in all the other routes? 
`var body: [String: Any] = [:]`

File: Sources/StripeKit/Core Resources/PaymentIntents/PaymentIntent.swift [hunk 2]
```diff
@@ -76,11 +76,16 @@ public struct StripePaymentIntent: StripeModel {
     /// Status of this PaymentIntent, one of `requires_payment_method`, `requires_confirmation`, `requires_action`, `processing`, `requires_capture`, `canceled`, or `succeeded`.
     public var status: StripePaymentIntentStatus?
     /// The data with which to automatically create a Transfer when the payment is finalized. See the PaymentIntents Connect usage guide for details.
-    public var transferData: [String: String]?
+    public var transferData: TransferData?
     /// A string that identifies the resulting payment as part of a group. See the PaymentIntents Connect usage guide for details.
     public var transferGroup: String?
 }
 
+public struct TransferData: StripeModel {
```
Comment (Andrewangeta) [inline Sources/StripeKit/Core Resources/PaymentIntents/PaymentIntent.swift]: Can we prefix this types name with `StripePaymentIntent` so it’ll be `StripePaymentIntentTransferData`. This way it’ll be clear when using this type what it’s associated with. 

Also let’s add some [documentation](https://stripe.com/docs/api/payment_intents/object#payment_intent_object-transfer_data) above each property so their usage can be clear.

Comment (Maxim-Inv) [reply Sources/StripeKit/Core Resources/PaymentIntents/PaymentIntent.swift]: It makes sense.

File: Sources/StripeKit/Connect/Accounts/AccountRoutes.swift (hunk 1 above)
Comment (Maxim-Inv) [reply Sources/StripeKit/Connect/Accounts/AccountRoutes.swift]: Ok

File: Sources/StripeKit/Core Resources/PaymentIntents/PaymentIntent.swift [hunk 3]
```diff
@@ -76,11 +76,18 @@ public struct StripePaymentIntent: StripeModel {
...
     /// The data with which to automatically create a Transfer when the payment is finalized. See the PaymentIntents Connect usage guide for details.
-    public var transferData: [String: String]?
+    public var transferData: StripePaymentIntentTransferData?
     /// A string that identifies the resulting payment as part of a group. See the PaymentIntents Connect usage guide for details.
     public var transferGroup: String?
 }
 
+public struct StripePaymentIntentTransferData: StripeModel {
+	/// Amount intended to be collected by this PaymentIntent. A positive integer representing how much to charge in the smallest currency unit
+	var destination: String?
```
Comment (Andrewangeta) [inline Sources/StripeKit/Core Resources/PaymentIntents/PaymentIntent.swift]: Can we mark these properties as  `public`.

Comment (Maxim-Inv) [reply Sources/StripeKit/Core Resources/PaymentIntents/PaymentIntent.swift]: Of course

**Comment by Maxim-Inv** [comment]
@Andrewangeta could you accept this PR?

**Comment by Andrewangeta** [comment]
Yep approving now. Gonna merge and tag along with the stripe header PR.

**Comment by Andrewangeta** [comment]
this fixes #66

**Comment by Maxim-Inv** [comment]
Thanks

**Pull request merged**